go/vt/srvtopo/resilient_server.go

```diff
@@ -206,16 +206,24 @@ type srvKeyspaceEntry struct {
 
 // NewResilientServer creates a new ResilientServer
 // based on the provided topo.Server.
-func NewResilientServer(base *topo.Server, counterPrefix string) *ResilientServer {
+func NewResilientServer(base *topo.Server, counterPrefix string, publishMetrics bool) *ResilientServer {
```

Comment:
I looked at all use cases of NewResilientServer. `counterPrefix` is always set. So, we could say that an empty `counterPrefix` means "don't publish".